_api-reference/cat/cat-nodes.md

```diff
@@ -17,25 +17,36 @@ The CAT nodes operation lists node-level information, including node roles and l
 A few important node metrics are `pid`, `name`, `cluster_manager`, `ip`, `port`, `version`, `build`, `jdk`, along with `disk`, `heap`, `ram`, and `file_desc`.
 
 
+<!-- spec_insert_start
+api: cat.nodes
+component: endpoints
+-->
 ## Endpoints
-
 ```json
-GET _cat/nodes
+GET /_cat/nodes
 ```
+<!-- spec_insert_end -->
 
-## Query parameters
-
-All CAT nodes URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameters:
-
-Parameter | Type | Description
-:--- | :--- | :---
-bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
-full_id | Boolean | If true, return the full node ID. If false, return the shortened node ID. Defaults to false.
-cluster_manager_timeout | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
-time | Time | Specify the units for time. For example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
-include_unloaded_segments | Boolean | Whether to include information from segments not loaded into memory. Default is `false`.
+<!-- spec_insert_start
+api: cat.nodes
+component: query_parameters
+columns: Parameter,Type,Description,Default
+include_deprecated: false
+-->
+## Query parameters
+Parameter | Type | Description | Default
+:--- | :--- | :--- | :---
+`bytes` | String | The units used to display byte values. | 
+`cluster_manager_timeout` | String | The amount of time allowed to establish a connection to the cluster manager node. | 
+`format` | String | A short version of the `Accept` header, such as `json` or `yaml`. | 
+`full_id` | Boolean or String | When `true`, returns the full node ID. When `false`, returns the shortened node ID. | `false`
+`h` | List | A comma-separated list of column names to display. | 
+`help` | Boolean | Return help information. | `false`
+`s` | List | A comma-separated list of column names or column aliases to sort by. | 
+`time` | String | Specifies the time units, for example, `5d` or `7h`. For more information, see [Supported units](https://opensearch.org/docs/latest/api-reference/units/). | 
```

Comment:
The previous instance of `time` did not include examples. I think examples are nice and a link to the units should be provided.

Reply:
Are all `time` parameter descriptions coming from different places in the spec? If so, we should create a reference variable in the spec and source all descriptions from that variable.

Reply:
Let's talk about using `$ref` for descriptions in another venue. Using `$refs` for descriptions also affects how other developers contribute new or missing specs, so I think it warrants a larger discussion about how to do it properly.

Reply:
We talked about this in the API spec repo a while ago https://github.com/opensearch-project/opensearch-api-specification/pull/726#issuecomment-2536951766

Reply:
Yes we did. When I tried to do this locally it the spec would error out, similar to what's discussed in this issue: https://github.com/opensearch-project/opensearch-api-specification/issues/577. I believe that you would have to create a separate schema which includes only the shared description, as opposed to referencing an already used description to another schema. I'll make an issue the api-specification repo detailing what I tried and encountered.

Reply:
I think we just need to keep the definitions in the common schema and generate valid values off of the enum (I opened an issue in the api-spec repo https://github.com/opensearch-project/opensearch-api-specification/issues/783) cc: @nhtruong